prototypes/examples/create-react-app/public/index.html

```diff
@@ -109,6 +110,18 @@
     <div id="root"></div>
 
     <!-- webpack runtime script -->
```

Comment:
we can patch webpack to not need this if using native modules and exclusively using webbundles, right? that'd be a nice feature to show too in a future PR :)

Reply:
Basically, webpack takes the `index.html` template and injects links to its script files and some inline code. We don't need the former (they will be added after the bundle arrives) but we do need the latter.

I think that it is something that can be fixed in the webpack configuration.